core/src/main/java/nl/nn/adapterframework/processors/InputOutputPipeProcessor.java

```diff
@@ -103,6 +103,7 @@ protected PipeRunResult processPipe(PipeLine pipeLine, IPipe pipe, Message messa
 			if (pipe instanceof FixedForwardPipe) {
 				FixedForwardPipe ffPipe = (FixedForwardPipe) pipe;
 				if (ffPipe.skipPipe(message, pipeLineSession)) {
+					log.info("skipping pipe [{}] because of empty input", pipe::getName);
```

Comment:
Het log statement op deze plek klopt niet helemaal met de controle dat uitgevoerd is in de `skipPipe` methode..

De skip on empty & is not empty check is slechts één van de meerdere redenen waarom deze methode `true` zou kunnen retourneren..

Reply:
Klopt ik zag het ook. Ben er nog niet helemaal over uit of ik een nieuwe processor wil maken of überhaupt de code wil verplaatsen. 
Het zit nu iig best raar in elkaar. Voor de 7.9 en 8.0 moeten we denk ik de log regel verplaatsen naar daar waar de checks worden gedaan.